crates/lair_keystore_api/src/config.rs

```diff
@@ -238,7 +238,7 @@ impl LairServerConfigInner {
             // on not-windows, we default to using unix domain sockets
             #[cfg(not(windows))]
             let connection_url = {
-                let mut con_path = root_path.clone();
+                let mut con_path = root_path.clone().canonicalize().unwrap();
```

Comment:
Please switch to using [dunce::canonicalize](https://docs.rs/dunce/latest/dunce/fn.canonicalize.html). I know this is a non-windows code path, but sometimes code gets copied and the std canonicalize is a big no-no on windows.